site/public/js/ta-grading-rubric.js

```diff
@@ -1270,19 +1270,33 @@ function getGradedComponentFromDOM(component_id) {
         verifier_id: dataDOMElement.attr('data-verifier_id')
     };
 }
-
 /**
  * Gets the scores data from the DOM (auto grading earned/possible and ta grading possible)
  * @return {Object}
  */
 function getScoresFromDOM() {
+    let gradeable_id = getGradeableId();
     let dataDOMElement = $('#gradeable-scores-id');
-
-    // Get the TA grading scores
+    var user_group_txt = $.ajax({
```

Comment:
Any reason why this (and lines 1291 and 1292) are marked `var` instead of `let`?

Reply:
Also, does this need custom functionality or would a standard JQuery GET request be simpler?

Reply:
I'm not sure how I would simplify it

Reply:
Would [this](https://api.jquery.com/jquery.get/) serve the same purpose here/be simpler?